cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -44,7 +44,7 @@ describe('Comment on articles', () => {
   describe('Comments using mention autocomplete', () => {
     it('should comment on an article with user mention autocomplete suggesting max 6 users', () => {
       cy.intercept(
-        { method: 'GET', url: '/search/usernames' },
+        { method: 'GET', url: '/search/usernames?username=se' },
```

Comment:
Questions: Is this new search API changes?

Reply:
No, before Cypress `intercept` would match a url with any query param, e.g. `intercept('/search/usernames)` would match `/search/usernames?username=s`, `/search/usernames?username=se`, `/search/usernames?someOtherParam=xxx`, but now you have to be specific to the full URL, including any query